data/saas/dataset/mandrill_dataset.yml

```diff
@@ -0,0 +1,5 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: Mandril Dataset
+    description: A sample dataset representing the Mandril connector for Fides
+    collections: []
```

Comment:
The dataset here is a placeholder to borrow Adam's terminology.  I think the fastest path forward with this is making Mandrill a saas connector so we can still utilize the same connector workflow + database tables.

So Mandrill here doesn't need collections outlined, but saas connectors that have both access and consent for example, would outline collections.